custom-connectors/Snowflake/script.csx

```diff
@@ -1,123 +1,263 @@
-                {
-                    var name = col["name"].ToString();
-                    string type = col["type"].ToString();
-                    if (newRow.ContainsKey(name)) name = name + "_" + Convert.ToString(i);
-
-                    switch (type)
-                    {
-                        case "fixed":
-                            long myLong = long.Parse(row[i].ToString());
-                            newRow.Add(new JProperty(name.ToString(), myLong));
-                            break;
-                        case "float":
-                            float myFloat = float.Parse(row[i].ToString());
-                            newRow.Add(new JProperty(name.ToString(), myFloat));
-                            break;
-                        case "boolean":
-                            bool myBool = bool.Parse(row[i].ToString());
-                            newRow.Add(new JProperty(name.ToString(), myBool));
-                            break;
-                        default:
-                            if (type.ToString().IndexOf("time") >= 0)
-                            {
-
-                                double unixTimeStamp = Convert.ToDouble(row[i].ToString());
-
-                                DateTime dateTime = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
-                                dateTime = dateTime.AddSeconds(unixTimeStamp);
-
-                                newRow.Add(new JProperty(name.ToString(), dateTime.ToUniversalTime().ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fff'Z'")));
-                            }
-                            else
-                            {
-                                var val = row[i];
-                                newRow.Add(new JProperty(name.ToString(), val));
-                            }
-                            break;
-                    }
-                    i++;
-
-                }
-
-                newRows.Add(newRow);
-
-            }
-
-            JObject output = new JObject
-            {
-                ["data"] = newRows
-            };
-            response = new HttpResponseMessage(HttpStatusCode.OK);
-            response.Content = CreateJsonContent(output.ToString());
-            return response;
-        }
-        catch (JsonReaderException ex)
-        {
-            return createErrorResponse("'rowType' or 'data' are in an invalid format: " + ex, HttpStatusCode.BadRequest);
-        }
-        catch (Exception ex)
-        {
-            return createErrorResponse(ex.GetType()+":"+ex, HttpStatusCode.InternalServerError);
-        }
-    }
-
-    private static HttpResponseMessage createErrorResponse(String msg, HttpStatusCode code)
-    {
-        JObject output = new JObject
-        {
-            ["Message"] = msg
-        };
-        var response = new HttpResponseMessage(code);
-        response.Content = CreateJsonContent(output.ToString());
-        return response;
-    }
-
-}
\ No newline at end of file
+

```

Comment:
@marzillo-snow all files should be properly formatted. please fix the file indentation.